src/Forte.Optimizely.ContentUsage/Api/Features/ContentType/ContentTypDtoBuilder.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using EPiServer.DataAbstraction;
+
+namespace Forte.Optimizely.ContentUsage.Api.Features.ContentType;
+
+public class ContentTypeDtoBuilder
+{
+    private readonly ContentTypeUsagesRepository _contentTypeUsagesRepository;
+
+    public ContentTypeDtoBuilder(ContentTypeUsagesRepository contentTypeUsagesRepository)
+    {
+        _contentTypeUsagesRepository = contentTypeUsagesRepository;
+    }
+
+    public async Task<ContentTypeDto> Build(EPiServer.DataAbstraction.ContentType contentType,
+        CancellationToken cancellationToken)
+    {
+        var dto = new ContentTypeDto
+        {
+            ID = contentType.ID,
+            DisplayName = contentType.DisplayName,
+            Name = contentType.Name,
+            Guid = contentType.GUID,
+            Type = contentType.Base.ToString(),
+            Statistics = new UsageStatisticDto[] { }
```

Comment:
To avoid unnecessary allocation:
```suggestion
            Statistics = Array.Empty<UsageStatisticDto>()
```
or initialize this prop with data from line 35-38